src/main/java/com/google/sps/data/Post.java

```diff
@@ -186,34 +252,44 @@ public void entityToPost(Entity entity) {
     timeSort = Integer.parseInt(entity.getProperty("timeSort").toString());
     collegeId = (String) entity.getKind();
     postId = entity.getKey().toString();
+    imageServingUrl = (String) entity.getProperty("imageServingUrl");
+    blobKey = null; // Does not set the blobKey. It can be used to change the blob, so it should not be sent to the client side.
```

Comment:
I like that you explicitly call out that we shouldn't pass the blobKey back to the client!

Reply:
Is there a reason that this comment is inline instead of before?

Reply:
Putting it above is better.